gateway/src/main/java/com/lyft/data/gateway/router/impl/GatewayBackendManagerImpl.java

```diff
@@ -131,7 +128,7 @@ private void reloadClusterStateAtStartUp() {
             }
           });
     } catch (Exception e) {
-      log.error("Could not read previous backend cluster state", e);
+      log.warn("No previous backend cluster state found - " + e.getMessage());
```

Comment:
Would the exception stacktrace still be helpful in debugging ?

Reply:
The reason for this change is - https://github.com/lyft/presto-gateway/issues/53 
Logging complete stacktrace for a harmless warning is confusing.